concepts/05 UI Components/Map/03 Accessibility/10 Accessibility Standards Compliance.md

```diff
@@ -7,8 +7,13 @@ The Map component meets a variety of <a href="https://www.access-board.gov/ict/"
         <th>Exception description</th>
     </tr>
     <tr>
-        <td>-</td>
+        <td>501 (Web)(Software) <br> 504.2 (Authoring Tool) <br> 602.3 (Support Docs)</td>
+        <td>1.3.1 Info and Relationships (Level A)</td>
+        <td>Map does not support Windows High Contrast themes.</td>
```

Comment:
Do we really wanna describe such an exception for each component? I'm totally not in a context, but it can be not actual for map as the whole UI is just loaded from a provider